src/test/java/seedu/saveit/logic/commands/SortCommandTest.java

```diff
@@ -76,10 +80,18 @@ public void execute_sortAfterUpdate_success() {
         model.addIssue(issue);
         SortCommand command = new SortCommand(issueSort);
         assertCommandSuccess(command, model, commandHistory, expectedMessage, expectedModel);
-        assertEquals(Arrays.asList(DANIEL, BENSON, ALICE, issue, GEORGE, FIONA, ELLE, CARL),
+        assertEquals(Arrays.asList(DANIEL, BENSON, ALICE, CARL, ELLE, FIONA, GEORGE, issue),
                 model.getFilteredAndSortedIssueList());
     }
 
+    @Test
+    public void execute_notUnderRootLevel_failure() {
+        model.resetDirectory(Directory.formDirectory(INDEX_FIRST_ISSUE.getOneBased(), 0));
```

Comment:
I think maybe use a new class (e.g. DirectoryBuilder) in test util will be more unified

Reply:
Yep exactly. Once your PR is merged I will updated this one. I noticed you created alr so didn't bother to create lol.

Reply:
okok